pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -843,7 +864,7 @@ type KubeVirtInfraStorageClass struct {
 	Name string `json:"name"`
 	// Optional: IsDefaultClass. If true, the created StorageClass in the tenant cluster will be annotated with:
 	// storageclass.kubernetes.io/is-default-class : true
-	// If missing or false, annotation will be:
+	// If missing or false,  annotation will be:
```

Comment:
Did you mean to insert a "the" here to fix the grammar?

Reply:
Interesting, I don't see this change in my revision 🤔